components/profile_popover/profile_popover.tsx

```diff
@@ -710,21 +758,56 @@ ProfilePopoverState
             title = this.props.overwriteName;
             roleTitle = '';
         } else if (this.props.hasMention) {
-            title = <a onClick={this.handleMentionKeyClick}>{title}</a>;
+            title = (
+                <button
+                    className='style--link user-popover__username'
+                    onClick={this.handleMentionKeyClick}
+                >
+                    {title}
+                </button>);
+        } else {
+            title = <span className='user-popover__username'>{title}</span>;
         }
         title = (
             <span data-testid={`profilePopoverTitle_${this.props.user.username}`}>
-                <span className='user-popover__username'>{title}</span>
+                {title}
                 {roleTitle}
             </span>
         );
+
+        const displayName = displayUsername(this.props.user, this.props.teammateNameDisplay);
+
+        const tabCatcher = (
+            <span
+                tabIndex={0}
+                onFocus={(e) => (e.relatedTarget as HTMLElement).focus()}
+            />
+        );
+
         return (
             <Popover
                 {...popoverProps}
-                title={title}
                 id='user-profile-popover'
             >
-                {dataContent}
+                {tabCatcher}
+                <div
+                    role='dialog'
+                    aria-label={Utils.localizeAndFormatMessage('profile_popover.profileLabel', 'Profile for {name}', {name: displayName})}
+                    onKeyDown={this.handleKeyDown}
+                    className={A11yClassNames.POPUP}
+                >
+                    <div
+                        tabIndex={-1}
+                        className='popover-title'
+                        ref={this.titleRef}
```

Comment:
The static header is actually the first thing that gets focused inside the profile popover, rather than the first button/link. This is perfectly fine for accessibility, but after the UI update we will want to focus the close button first.